### PR TITLE
chore: add DpProgressBar to Storybook.

### PR DESCRIPTION
**Pull request by ahmad-demos**
- add DpProgressBar to Storybook.

### CODE REVIEW

File: src/components/DpProgressBar/DpProgressBar.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  label="LabelText"
+  percentage="someNumber" />
+```
+
+## Indeterminate ProgressBar
+
+Use the indeterminate ProgressBar if you don't know how long a task will take. It shows a cyclic animation instead of
+linear progress.
+
+The blue color indicates to the users that something is happening, but we do not have information on
```
Comment (spiess-demos) [inline src/components/DpProgressBar/DpProgressBar.stories.mdx:31]:
```suggestion
The blue color indicates that something is happening, but we do not have information on
```

File: src/components/DpProgressBar/DpProgressBar.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  label="LabelText"
+  indeterminate />
+```
+
+## When to use
+Use this component if you want to communicate to the user
+
+**a)** the progress of a task or process (determinate)
+
+**b)** the information that a process is running (indeterminate).
```
Comment (spiess-demos) [inline src/components/DpProgressBar/DpProgressBar.stories.mdx:45]: Let's also add that for loading, we have DpLoading or DpSkeletonBox. Maybe we could even link to the documentation of those?

```suggestion
## When to use

Use this component if you want to communicate to the user

**a)** the progress of a task or process (determinate)

**b)** the information that a process is running (indeterminate).

Do not use it as a loading indicator. To indicate loading state, use DpLoading or DpSkeletonBox.

```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/DpProgressBar/DpProgressBar.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## When to use
+
+Use this component if you want to communicate to the user
+
+**a)** the progress of a task or process (determinate)
+
+**b)** the information that a process is running (indeterminate).
+
+Do not use it as a loading indicator. To indicate loading state, use DpLoading or DpSkeletonBox.
```
Comment (spiess-demos) [inline src/components/DpProgressBar/DpProgressBar.stories.mdx:48]: Could you add links to the documentation of DpLoading and DpSkeletonBox here?

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: src/components/DpProgressBar/DpProgressBar.stories.mdx (hunk 3 above)
Comment (ahmad-demos) [reply src/components/DpProgressBar/DpProgressBar.stories.mdx]: Done in https://github.com/demos-europe/demosplan-ui/pull/531/commits/0dbec27e6f723d5158f7c37ff9382f3e9fe884dd
thakns!

**Pull request merged**